src/SIL.XForge.Scripture/Services/ParatextSyncRunner.cs

```diff
@@ -1668,16 +1668,6 @@ private void Log(string message, string? projectSFId = null, string? userId = nu
             LogMetric(message);
         }
 
-        private void LogMetric(string message)
-        {
-            _syncMetrics.Log.Add($"{DateTime.UtcNow} {message}");
-            _syncMetricsRepository
-                .ReplaceAsync(_syncMetrics, true)
-                .ContinueWith(task =>
-                {
-                    if (!task.Result)
-                        _logger.LogInformation("The sync metrics could not be updated in MongoDB");
-                });
-        }
+        private void LogMetric(string message) => _syncMetrics.Log.Add($"{DateTime.UtcNow} {message}");
```

Comment:
You didn't want to log to stdout?

Reply:
Logging to stdout was done as a hotfix to live, and I also made this change on master. So I was thinking we would revert the change on master completely, and then if we decided we wanted to log to stdout we could chery-pick that change from live.